geode-core/src/main/java/org/apache/geode/internal/admin/remote/DistributionLocatorId.java

```diff
@@ -305,7 +354,11 @@ public String toString() {
    * Indicates whether some other object is "equal to" this one.
    *
    * @param other the reference object with which to compare.
-   * @return true if this object is the same as the obj argument; false otherwise.
+   *
+   *        If member name is defined in both objects, and both objects have same member name,
```

Comment:
this javadoc change no longer applies since the equals() method no longer checks member names